gds/samples/python/sample_002.py

```diff
@@ -0,0 +1,75 @@
+"""
+ Copyright 2025 NVIDIA Corporation.  All rights reserved.
+ 
+ Please refer to the NVIDIA end user license agreement (EULA) associated
+ with this source code for terms and conditions that govern your use of
+ this software. Any use, reproduction, disclosure, or distribution of
+ this software and related documentation outside the terms of the EULA
+ is strictly prohibited.
+"""
+"""
+ Sample KvikIO write test using context manager.
+
+ This test writes data from GPU memory to a file using write and raw_write.
+ The opening and closing of the file is handled by a context manager.
+ For verification, input data has a pattern.
+ User can verify the output file-data after write using
+ hexdump -C <filepath>
+ 00000000  ab ab ab ab ab ab ab ab  ab ab ab ab ab ab ab ab  |................|
+ *
+ 00020000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+ *
+ 00021000  ab ab ab ab ab ab ab ab  ab ab ab ab ab ab ab ab  |................|
+ *
+ 00041000
+"""
+
+import sys
+import cupy
+import kvikio
+
+# Constants
+FILE_SIZE_BYTES = (128 * 1024) # 128 KB
+FILE_OFFSET_BYTES = (4 * 1024) # 4 KB
+
+def main(path: str) -> None:
+    """
+    Writes data to a file using KvikIO.
+
+    Args:
+        path (str): The path to the file to write to.
+    """
+    
+    print(f"Creating data vector of size {FILE_SIZE_BYTES} bytes")
+    buf = cupy.full((FILE_SIZE_BYTES,), int("ab", 16), dtype=cupy.uint8)
+    print(f"Opening file: {path}")
+    with kvikio.CuFile(path, "w") as file_writer:
+        print("Writing data vector to file")
+        # write is a blocking call implemented by calling pwrite and waiting for 
+        # the IO to complete before returning to the caller.
+        # like pwrite, it uses an internal threadpool on top of the cufile library.
+        # It supports host and device memory.
+        ret = file_writer.write(buf)
+        if ret < 0:
+            print(f"Error writing file: {ret}")
+            return
```

Comment:
```suggestion
        if ret < 0:
            print(f"Error writing file: {ret}")
            return
```
I don't think you need this check. In fact, `ret` is a `unsigned int` so it will never be negative.

Reply:
Thanks Mads, from the docs I thought it was just an int and could have sworn I remembered it being negative when there was an error. But if that's not the case I will go ahead and remove these checks.

Reply:
It shouldn't be negative, but of course there could be bugs :)